docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -228,13 +228,23 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 When implementing a `org.ehcache.spi.serialization.Serializer`, you obviously need to respect the interface contract.
 In addition, there are a few implicit contracts:
 
-* Your implementation must have a public constructor taking a `ClassLoader` as parameter.
-* There are two annotations that you have to use to mark your `Serializer` - a single implementation can use both:
-** `org.ehcache.spi.serialization.Serializer.Transient`: With this annotation you indicate that the `Serializer`
-  implementation is fit for use inside a transient cache.
-** `org.ehcache.spi.serialization.Serializer.Persistent`: With this annotation you indicate that the `Serializer`
-  implementation is fit for use inside a persistent cache. The constructor can take an additional parameter, a
-  `org.ehcache.spi.service.FileBasedPersistenceContext` which can then be used to persist serialization metadata.
+* A serializer implementation to be used with transient caches must have a public constructor taking a `ClassLoader` as parameter.
+* A serializer implementation to be used with persistent caches must have a public two-argument constructor taking a `ClassLoader` and a `FileBasedPersistenceContext` as parameters.
+ 
+You are free to have as many extra constructors as you want. But these are the only constructors that will be used by
+ehcache to instantiate these serializers. For a transient cache the single-argument constructor will be used and for a
+persistent cache the two-argument constructor will be used. Even though the user is enforced to have either or both the
+constructors in his serializer implementation, the usage of these arguments is upto the user. The user can decide to
+completely ignore these parameters. But the presence of these constructors is a strict requirement.
+ 
+If you decide to use the `FileBasedPersistenceContext` parameter there are some extra perks. The cache manager will keep
+the serializer's persistent data at the same space as the cache's data is stored. There is no added advantage in doing
+so. It is just an option. If you do not want the data to co-exist for some reason then you can choose to ignore the 
+`FileBasedPersistenceContext` altogether and persist the data wherever you want.
+
+The constructor requirements detailed above are applicable only if you are using the `DefaultSerializationProvider`
+service provided by ehcache(That would be the case for most of the users, unless you are using your own implementation
```

Comment:
You can drop parenthesis here and have the bit between them as the last sentence.